sympy/solvers/tests/test_pde.py

```diff
@@ -231,4 +231,4 @@ def test_pdsolve_variable_coeff():
 
     eq = exp(2*x)*(u.diff(y)) + y*u - u
     sol = pdsolve(eq, hint='1st_linear_variable_coeff')
-    assert sol == Eq(u, exp((-y**2 + 2*y + 2*F(x))*exp(-2*x)/2))
+    assert sol == Eq(f(x, y), F(x)*exp(-y*(y - 2)*exp(-2*x)/2))
```

Comment:
This should still be `u` rather than `f(x, y)` to keep it consistent with the examples above